pkg/store/errors/errors.go

```diff
@@ -21,4 +21,5 @@ var (
 	ErrAmbiguous          = errors.New("ambiguous query: more than one result returned")
 	ErrInternal           = errors.New("something critical went wrong, please try again later")
 	ErrDaybreakHasTxns    = errors.New("daybreak counterparty will not be deleted because it has transactions")
+	ErrInvalidString      = errors.New("string is not valid")
```

Comment:
```suggestion
	ErrNullString      = errors.New("cannot apply string method to a null value")
```

`Valid` in a `sql.NullString` refers to the fact that it `IsNull` -- semantically this is important because it is different from empty string (since Go doesn't have a None type).